recipes/server_setup.rb

```diff
@@ -114,37 +121,47 @@
       end
 
       # Create option string
+      force = false
       options = ''
       case volume_values['volume_type']
       when 'replicated'
-        # Ensure the trusted pool has the correct number of bricks available
-        if brick_count < volume_values['replica_count']
-          Chef::Log.warn("Correct number of bricks not available: #{brick_count} available, #{volume_values['replica_count']} required for volume #{volume_name}. Skipping...")
-          next
-        else
-          options = "replica #{volume_values['replica_count']}"
-          volume_bricks.each do |peer, vbricks|
-            options << " #{peer}:#{vbricks.first}"
-          end
-        end
+        # Replicated can be anything from a single node to X nodes. Replica_count should equal number of bricks.
+        Chef::Log.warn('You have specified replicated, so the attribute replica_count will be set to be the same number as the bricks you have')
+        node.set['gluster']['server']['volumes'][volume_name]['replica_count'] = brick_count
+        options = "replica #{brick_count}"
       when 'distributed-replicated'
-        # Ensure the trusted pool has the correct number of bricks available
-        required_bricks = (volume_values['replica_count'] * volume_values['peers'].count)
-        if brick_count != required_bricks
-          Chef::Log.warn("Correct number of bricks not available: #{brick_count} available, #{required_bricks} required for volume #{volume_name}. Skipping...")
+        # This must be at least replica_count * 2
+        required_bricks = (volume_values['replica_count'] * 2)
```

Comment:
`The number of bricks must be a multiple of the replica count`
http://gluster.readthedocs.org/en/latest/Quick-Start-Guide/Architecture/

`brick_count % volume_values['replica_count'] == 0`